packages/docs/docs/getting-started/code-block.mdx

```diff
@@ -0,0 +1,249 @@
+---
+sidebar_position: 12
+slug: /code-block
+---
```

Comment:
Could you extract this, together with the Latex [component docs from here](https://github.com/motion-canvas/motion-canvas/blob/main/packages/docs/docs/getting-started/nodes.mdx), and put it in a separate top-level category called nodes?
So we'd have something like this:
```
Getting Started
Available Nodes
Advanced
```

Reply:
I called in "Components", as my gut reflex was that that is a more commonly known term. Just let me know if you want "Advanced Nodes" and I'll change it back. I also move Custom Components into the end of the category, as that seemed to fit. And I fixed the order of the other files, since I was reordering them all anyways.